custom_components/waste_collection_schedule/waste_collection_schedule/source/birmingham_gov_uk.py

```diff
@@ -0,0 +1,95 @@
+import re
+from datetime import datetime
+from dateutil.parser import parse
+from dateutil.relativedelta import relativedelta
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection
+
+TITLE = "Birmingham City Council"
+DESCRIPTION = "Source for birmingham.gov.uk services for Birmingham, UK."
+URL = "https://birmingham.gov.uk"
+TEST_CASES = {
+    "Cherry Tree Croft": {"uprn": "100070321799", "postcode": "B27 6TF"},
+    "Ludgate Loft Apartments": {"uprn": "10033389698", "postcode": "B3 1DW"},
+    "Windermere Road": {"uprn": "100070566109", "postcode": "B13 9JP"},
```

Comment:
Please make at least one test case test an integer UPRN (users should not worry about whether their input is handled as a string or an integer, and we had problems with this in the past) (also Your example in the md file is read as an integer)

Reply:
Updated.